backend/src/api/interview.js

```diff
@@ -0,0 +1,129 @@
+const express = require('express')
+const router = express.Router()
+const { errorWrap } = require('../middleware')
+const { Interview } = require('../models')
+var mongodb = require('mongodb')
+
+router.get(
+  '/verified-interviewer',
+  errorWrap(async (req, res) => {
+    // const foundKey = await Interview.findById(req.params.interviewerKey);
+    // res.json({ result: foundKey })
+    /*** NOT IMPLEMENTED */
+  })
+)
+
+router.get(
+  '/',
+  errorWrap(async (req, res) => {
+    let interviews
+    interviews = await Interview.find()
+    res.json({ result: interviews })
+  })
+)
+
+router.get(
+  '/:interview_id',
```

Comment:
generally, I would put anything that has a parameter at the end just because frameworks use regex to match urls. since `/:interview_id` is a catch all function for any url with a string after `interviews` (eg `/interviews/10294809570`), say you had another endpoint `/interviews/questions` that you defined **after** this catch all function. Then if you actually tried to get `/interviews/questions`, express would think that you are look for an Interview with id `questions` because it went to the catch all function. 

Not sure if I explained it well, but the following is bad. 
```Javascript
router.get('/:interview_id', (req,res) => {
    console.log(req.params.interview_id)
}
// this function should be written before '/:interview_id'
router.get('/question', (req,res) => {
 .....
}
```

Reply:
the way you're doing it right now is fine. just watch out for that, it will save you an hour of debugging.